esmvaltool/diag_scripts/regional_downscaling/Figure9.39.ncl

```diff
@@ -0,0 +1,783 @@
+; ###########  ###############################################################
+; # SEASONAL OR ANNUAL BIAS                                                #
+; #############################################################################
+; # Author: Irene Cionni(ENEA, Italy)                                         #
+; # CRESCENDO project
+; #############################################################################
+;
+; # Description:                                                              #
+; #    * Create seasonal or annual bias box plots at selected regions         #
+; #       - Read regions
+; #        -create mask map using region poligons
+; #       - for all the models calculate the bias respect @refModel
+; #       - sort biases for each selected project&experiment&mip
+; #       - evaluate percentiles (5th 25th 50th 75th 95th)
+; #       - plot the first selected project&experiment&mip as box-and-whisker
+; #       - plot the others selected projects&experiment&mip as markers
+; #       - compare the root-mean-square error of first project&experiment&mip
+; #         with the second if exist  ibelled with redthe regions where
+; #         the first error is larger than the second
+; #
```

Comment:
corrected to: with the second if exist labeled with red for the regions where
; #         the first error is larger than the second.
Unfortunately, looking at that I realized, that the option is not working, because the NCL function "get_unique_difference" does not exist, which would be called in that case in Figure9.38.ncl (line 235), Figure9.39.ncl (line 192) and Figure9.40.ncl (line 202).
So far, I didn't find the function, it does not seem to be a standard NCL function. 
This means, at the moment the option to add a second multi model mean, e.g. for a second project does not work.

Reply:
I found it in ESMValTool v1 and will try if it works to include it into "regional_downscaling/regional_function.ncl"

Reply:
It runs when including these functions to regional_function.ncl